lightning-background-processor/src/lib.rs

```diff
@@ -2781,6 +2783,17 @@ mod tests {
 		);
 		let kv_store = Arc::new(KVStoreSyncWrapper(kv_store_sync));
 
+		// Yes, you can unsafe { turn off the borrow checker }
+		let lm_async: &'static LiquidityManager<_, _, _, _, _, _> = unsafe {
```

Comment:
This commit has a nice end result, but this - I am not sure how many devs understand it...